package.json

```diff
@@ -6,19 +6,22 @@
   "bugs": "https://github.com/forcedotcom/cli/issues",
   "dependencies": {
     "@oclif/config": "^1.17.0",
-    "@salesforce/command": "^3.0.4",
-    "@salesforce/core": "^2.15.2",
+    "@salesforce/command": "^3.0.5",
+    "@salesforce/core": "^2.15.4",
     "tslib": "^1"
   },
   "devDependencies": {
+    "@mshanemc/plugin-helpers": "2.1.0",
```

Comment:
@mshanemc has your plugin gone through 3pp approvals?

Reply:
sorry, that's my mistake.  The e2e testing stuff was not supposed to be part of the PR.  [we'll talk about that next week]

Reply:
removed...we have nuts now.